presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -412,6 +412,7 @@ private synchronized void startSplit(PrioritizedSplitRunner split)
     {
         allSplits.add(split);
         waitingSplits.offer(split);
+        log.debug("%s is started", split.toString());
```

Comment:
It's better to log the variable part at the end, since it makes the logs easier to read and search:
```java
log.debug("Started %s", split);
```
Also, don't call `toString()` as the logger will do that only if the level is enabled (thus it is free if disabled).

Reply:
Thanks!  I was just copying the finishing information 👍 
```
                            log.debug("%s is finished", split.getInfo());
```

But I agree use `Started...` is easier to search! :)